packages/website/docs/with-instantsearch.md

```diff
@@ -0,0 +1,51 @@
+---
+id: with-instantsearch
+title: Autocomplete with InstantSearch
+---
+
+Learn how to allow users to search from anywhere on your website, using Autocomplete with InstantSearch.
+
```

Comment:
Before jumping right into this, we should start with a quick intro about what InstantSearch is. Esp. if we are focusing on AC as a non-Algolia specific library, we can't assume all users will be familiar with InstantSearch.